src/jsonrpccpp/server/connectors/tcpsocketserver.cpp

```diff
@@ -8,7 +8,7 @@
  ************************************************************************/
 
 #include "tcpsocketserver.h"
-#ifdef __WIN32__
```

Comment:
It's `_WIN32` actually:
* https://msdn.microsoft.com/en-us/library/b0084kay.aspx